### DISCUSSION TITLE
How to validate the Mime Type?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17057] [Archive]
Tried to validate an image field, so that only images can be selected. None of these validation rules work for me.

`Image -> Mime Type Matches RegExp ^image\/*`
`Image -> Mime Type Starts with image/`
`Image -> Mime Type Starts with jpg`
`Image -> Mime Type Starts with JPG`

How to validate the Mime Type?

**Comment by martinsotirov** [comment]
Doesn't work for me either. @JonathanDoelan did you end up figuring it out?

**Reply by DanielBiegler** [reply]
The Image-Interface may be helpful here, see:

![image](https://github.com/directus/directus/assets/14810858/68689e53-9be9-45fa-a35a-8addf75f1b44)

It lets you select multiple formats too.

**Comment by DanielBiegler** [comment]
@JonathanDoelan Lets say you make a many-to-many relation on your table like so:

![image](https://github.com/directus/directus/assets/14810858/756d9db7-6c17-45ca-905a-8f6163da4298)

Then youre able to constrain the selection via the interface, see here

![image](https://github.com/directus/directus/assets/14810858/c966bb08-969b-4b51-bb2f-f1ad742c81d7)

Now your users can only select matching files, i.e. images - of course this is no real validation but this might help you already.
